backend/src/modules/player-profiles/presentation/rest-api/GoogleUserProfileRouter.ts

```diff
@@ -3,19 +3,19 @@ import { QueryPublisher } from '../../../../shared/core/application/query/QueryB
 import { StatusCodes } from 'http-status-codes';
 import { CommandPublisher } from '../../../../shared/core/application/command/CommandBus';
 import { DomainEventPublisher } from '../../../../shared/core/application/event/DomainEventBus';
-import { FindUserProfileById, FindUserProfileByIdResult } from '../../core/application/query/FindUserProfileById';
-import { CreateUserProfile } from '../../core/application/command/CreateUserProfile';
-import { PostUserProfileRequestBody } from './request/PostUserProfileRequestBody';
+import { FindGoogleUserProfileById, FindUserProfileByIdResult } from '../../core/application/query/FindGoogleUserProfileById';
+import { CreateGoogleUserProfile } from '../../core/application/command/CreateGoogleUserProfile';
+import { PostGoogleUserProfileRequestBody } from './request/PostGoogleUserProfileRequestBody';
 
-export function userProfileRouter(
+export function googleUserProfileRouter(
   commandPublisher: CommandPublisher,
   eventPublisher: DomainEventPublisher,
   queryPublisher: QueryPublisher,
 ): express.Router {
   const createUserProfile = async (request: Request, response: Response) => {
     const { userId } = request.params;
```

Comment:
userId musi byc odczytane z tokena w tym przypadku. Zeby nie bylo tak, ze kazdy sobie moze podmienic imie innej osoby. To bylo tak na szybko na potrzeby hackathonu :)